recipes/ghc-bootstrap/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
 test:
   commands:
     - ghc --help
-    - run-test.sh
+    - ./run-test.sh
```

Comment:
What I had in mind was to have a file called run_test.sh which will get automatically called by conda-build.
c.f: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-section

Reply:
In order to do what you wrote, you also need to add a test.files section to copy run-test.sh from the recipe dir.

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-files

Reply:
Ok, run_test.sh is now triggered automatically :-)